sdk/core/Azure.Core/src/HttpMessage.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
```

Comment:
Why is this added here?

Reply:
To enable `_errorCodes?.Contains(status)`

Reply:
I would use Span.Contains. 
I quickly run the following benchmark:
```csharp
public class MyClass
{
    int[] _array = new int[] { 1, 2, 3, 4, 5, 6 };

    [Benchmark]
    public bool Linq() => System.Linq.Enumerable.Contains(_array, 4);
        
    [Benchmark]
    public bool Find() => Array.IndexOf(_array, 4) >= 0;

    [Benchmark]
    public bool Span() => _array.AsSpan().Contains(4);
}
```

and here are the results
```
| Method |     Mean |     Error |    StdDev |
|------- |---------:|----------:|----------:|
|   Linq | 5.368 ns | 0.0189 ns | 0.0167 ns |
|   Find | 3.756 ns | 0.0144 ns | 0.0134 ns |
|   Span | 1.955 ns | 0.0122 ns | 0.0114 ns |